build/AzurePipelinesTemplates/WindowsAppSDK-SetupBuildEnvironment-Steps.yml

```diff
@@ -64,8 +64,8 @@ steps:
     inputs:
       targetType: filePath
       filePath: build\scripts\Install-WindowsSdkISO.ps1
-      arguments: 18362
-    displayName: 'Install Insider SDK (18362)'
+      arguments: 26100
+    displayName: 'Install Insider SDK (26100)'
```

Comment:
nit: not an Insider SDK - latest published version on aka.ms/windowssdk, for 24h2